### ISSUE TITLE
Debug: Draw tile offset is wrong in room 2+

### ISSUE THREAD

**Issue by segfault8** [tsoding/something#101]
Maybe due to padding between rooms?
If ROOM_PADDING is set to 0.0 the issue went away.

![draw-tile-issue](https://user-images.githubusercontent.com/64567823/82741177-d503e180-9d4f-11ea-825e-ab97ed03e98f.png)

**Comment by rexim** [comment]
Probably will get invalidated by #126

**Comment by kolumb** [comment]
Yes, since introduction of global grid (#129) debug overlay is right.
